### PR TITLE
Adds kubernetes audit logs config

### PR DESCRIPTION
**Pull request by marcinc**

### CODE REVIEW

File: conf.d/20_output_kubernetes_audit.conf [hunk 1]
```diff
@@ -0,0 +1,86 @@
+filter {
+  if "kubernetes" and "audit" in [tags] {
+    mutate {
+      split => { 
+        "message" => " " 
+      }
+    }
+
+    # Extract timestamp of the acutal event and overwrite @timestamp
```
Comment (vaijab) [inline conf.d/20_output_kubernetes_audit.conf]: `acutal` typo.

File: conf.d/20_output_kubernetes_audit.conf [hunk 2]
```diff
@@ -0,0 +1,86 @@
+filter {
+  if "kubernetes" and "audit" in [tags] {
+    mutate {
+      split => {
```
Comment (vaijab) [inline conf.d/20_output_kubernetes_audit.conf]: Maybe add a comment why you're splitting here?

File: conf.d/20_output_kubernetes_audit.conf [hunk 3]
```diff
@@ -0,0 +1,86 @@
+filter {
```
Comment (vaijab) [inline conf.d/20_output_kubernetes_audit.conf]: Should probably add `elasticsearch` to the file name.

**Review by vaijab — CHANGES_REQUESTED** [review]

File: conf.d/20_output_kubernetes_audit_elasticsearch.conf [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        add_tag => ["kubernetes_audit_filtered"]
+      }        
+    }
+  }
+}
+
+output {
+  if "kubernetes_audit_filtered" in [tags] {
+    elasticsearch {
+      index => "kubernetes-audit-%ELASTICSEARCH_INDEX_SUFFIX%%{+YYYY.MM.dd}"
```
Comment (vaijab) [inline conf.d/20_output_kubernetes_audit_elasticsearch.conf]: Should probably push to `kubernetes-` index and set document_type to `audit`.

**Review by vaijab — CHANGES_REQUESTED** [review]
If you could address one comment, otherwise looks good. Thanks!

File: conf.d/20_output_kubernetes_audit_elasticsearch.conf (hunk 4 above)
Comment (marcinc) [reply conf.d/20_output_kubernetes_audit_elasticsearch.conf]: Sure. Thought we wanted to separate that entirely but yeah... makes sense.

**Review by vaijab — APPROVED** [review]
LGTM

**Pull request merged**